lib/twilio/errors/generated.ts

```diff
@@ -252,6 +252,85 @@ export namespace GeneralErrors {
   }
 }
 
+export namespace MalformedRequestErrors {
+  export class MalformedRequestError extends TwilioError {
+    causes: string[] = [
+      'No CallSid in the message object.',
+      'No VoiceEventSid in the message object.',
+      'No payload in the message object.',
+      'Invalid or missing payload in the message object.',
+      'No message type in the message object.',
+    ];
+    code: number = 31100;
+    description: string = 'The request had malformed syntax.';
+    explanation: string = 'The request could not be understood due to malformed syntax.';
+    name: string = 'MalformedRequestError';
+    solutions: string[] = [
+      'Ensure the message object contains a valid CallSid.',
+      'Ensure the message object contains a valid VoiceEventSid.',
+      'Ensure the message object has a valid payload.',
+      'Ensure the message object has a valid message type.',
+    ];
+
+    constructor();
+    constructor(message: string);
+    constructor(error: Error | object);
+    constructor(message: string, error: Error | object);
+    constructor(messageOrError?: string | Error | object, error?: Error | object) {
+      super(messageOrError, error);
+      Object.setPrototypeOf(this, MalformedRequestErrors.MalformedRequestError.prototype);
+
+      const message: string = typeof messageOrError === 'string'
+        ? messageOrError
+        : this.explanation;
+
+      const originalError: Error | object | undefined = typeof messageOrError === 'object'
+        ? messageOrError
+        : error;
+
+      this.message = `${this.name} (${this.code}): ${message}`;
+      this.originalError = originalError;
+    }
+  }
+}
+
+export namespace AuthorizationErrors {
+  export class RateExceededError extends TwilioError {
```

Comment:
Did you run this on the browser? What does the error object look like? Please try to invoke it and post the object here.

Reply:
<img width="812" alt="Screen Shot 2022-10-21 at 1 49 39 PM" src="https://user-images.githubusercontent.com/12516091/197286795-959e6111-eb02-4f41-ad06-0d08081d9b2f.png">